specify/specify7
# refactor(FormEditor): use .getRelationship() in viewSpec

Refactor code in response to code changes in PR #4748 (it was merged before I had a chance to fix it)

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- Test that the thing #4748 was fixing is still fixed
- Test that an error is shown when you enter a name of the field that does not exist in the related table (see list of fields per table: https://sp7demofish.specifycloud.org/specify/data-model/)
- open forms with data entry
